cogs/plan.py

```diff
@@ -102,7 +102,7 @@ async def day(self,ctx: commands.context):
 
       
     msg = await ctx.send(embed=embed)
-    await msg.add_reaction("✅") #To get the emoji, in discord type "\:emoji_name:" then highlight with cursor and copy paste into code
+    await msg.add_reaction("✅") #To get the emoji, in nextcord type "\:emoji_name:" then highlight with cursor and copy paste into code
```

Comment:
This should remain as 'discord' as it is referring to the discord platform itself.